src/strategy/GGVStrategy.sol

```diff
@@ -18,7 +18,7 @@ contract GGVStrategy is Strategy {
     // ==================== Events ====================
```

Comment:
Nitpick: consider using the same natpsec and comments style as in StvPool and similar contracts. Just for consistency